runbotics/runbotics-orchestrator-ui/src/components/ResizableDrawer/ResizableDrawer.styles.scss

```diff
@@ -0,0 +1,8 @@
+body[data-dragmode] {
```

Comment:
hmm xd, please use styled components and avoid css files, lets discuss it on a call